source/technical/swift.rst

```diff
@@ -368,51 +368,64 @@ Mounting Object Store as a File System
 ======================================
```

Comment:
we can probably just delete this whole section

Reply:
we definitely can, but I'd be tempted to leave it as it's still a valid way to interact with the object store, and depending on how rclone mounts perform in "the wild" we may still have cases where people would/should interact via the swift client? I still think this section is good and valid docs, even if using the rclone mount is easier.

Reply:
I suggest it mostly because the `aws` client in particular is getting less and less compatible with non-aws S3 mounts, and was causing a bunch of issues

so I'd propose either removing it, or suggesting the rclone equivalents of the same operations

Reply:
Ok, I can remove it for now and if we need to start directing users to using rclone in this way then we can add docs on how to do it.

Reply:
done